internal/worker/dbaccessor/manifold.go

```diff
@@ -172,3 +179,15 @@ func dbAccessorOutput(in worker.Worker, out interface{}) error {
 	}
 	return nil
 }
+
+// IAASNodeManager returns a NodeManager that is configured to use
+// the cloud-local TLS terminated address for Dqlite.
+func IAASNodeManager(cfg agent.Config, logger Logger, slowQueryLogger coredatabase.SlowQueryLogger) NodeManager {
+	return database.NewNodeManager(cfg, false, logger, slowQueryLogger)
+}
+
+// CAASNodeManager returns a NodeManager that is configured to use
+// the loopback address for Dqlite.
+func CAASNodeManager(cfg agent.Config, logger Logger, slowQueryLogger coredatabase.SlowQueryLogger) NodeManager {
```

Comment:
This needs wiring up in the CAASManifolds.

Reply:
Done